subsys/bluetooth/host/att.c

```diff
@@ -3088,7 +3106,16 @@ int bt_eatt_connect(struct bt_conn *conn, uint8_t num_channels)
 		return -ENOMEM;
 	}
 
-	return bt_l2cap_ecred_chan_connect(conn, chan, BT_EATT_PSM);
+	while (created < i) {
+		err = bt_l2cap_ecred_chan_connect(conn, &chan[created], BT_EATT_PSM);
+		if (err < 0) {
+			return err;
+		}
+
+		created += 5; /* Max channels per request */
```

Comment:
```suggestion
		created += L2CAP_ECRED_CHAN_MAX; /* Max channels per request */
```

Reply:
Currently the macro is local to `l2cap.c`, but can move it to the internal header (and give it a non-misleading name).

Reply:
Please do :)